rm-datasource/src/main/java/io/seata/rm/datasource/undo/parser/FastjsonUndoLogParser.java

```diff
@@ -30,14 +30,15 @@
  * @author sharajava
  */
 @LoadLevel(name = FastjsonUndoLogParser.NAME)
-public class FastjsonUndoLogParser implements UndoLogParser {
+public class FastjsonUndoLogParser implements UndoLogParser, Initialize {
 
     public static final String NAME = "fastjson";
 
-    private static final SimplePropertyPreFilter FILTER = new SimplePropertyPreFilter();
+    private SimplePropertyPreFilter filter = new SimplePropertyPreFilter();
```

Comment:
'final' can be considered reserved, as are other classes.

Reply:
done